ndspy/bmg.py

```diff
@@ -36,12 +37,12 @@ class BMG:
     A class representing a BMG file.
     """
 
-    def __init__(self, data=None, *, id=0):
+    def __init__(self, data: bytes | None = None, *, id: int = 0) -> None:
```

Comment:
`None` is the only legal return type for `__init__` methods, so [MyPy lets you omit that annotation here](https://github.com/python/mypy/pull/5677). I just see it as clutter in this case, would you mind removing it?